Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -137,7 +137,7 @@ private async Task<string> CreateFleet(ComputeType computeType, string fleetLoca
             }
         }
 
-        public async Task<List<FleetAttributes>> ListFleetAttributes()
+        public async Task<List<FleetAttributes>> ListFleetAttributes(ComputeType computeType)
```

Comment:
It looks like this method was renamed in a separate PR that's causing merge conflicts